android/filament-android/src/main/java/com/google/android/filament/Skybox.java

```diff
@@ -21,6 +21,31 @@
 
 import com.google.android.filament.proguard.UsedByReflection;
 
+/**
+ * Skybox
+ * <p></p>
+ * When added to a Scene, the Skybox fills all untouched pixels.
+ * <p></p>
```

Comment:
In many places you are using the `<p></p>` pattern.  Instead of surrounding the line with two empty paragraphs, I think it's better to simply do:
 `<p>When added to a Scene, the Skybox fills all untouched pixels.</p>`

The line breaks should look fine when doing it this way. If you are unhappy with the size of the line break, that's not your fault, it's just a CSS issue. It's better to use HTML for semantics, not for style.

If you absolutely need an explicit line break, you can use one or more `<br>` tags; no need for a closing tag for that particular element -- HTML is not XML.